component/src/main/java/com/synopsys/integration/alert/component/certificates/CertificatesDescriptor.java

```diff
@@ -26,6 +26,11 @@ public class CertificatesDescriptor extends ComponentDescriptor {
     public static final String KEY_ALIAS = "alias";
     public static final String KEY_CERTIFICATE_CONTENT = "certificateContent";
 
+    // following fields are for client certificates
+    public static final String KEY_CLIENT_CERTIFICATE_CONTENT = "clientCertificateContent";
```

Comment:
This looks good as is to me.  I propose the question that since this is technically an MTLS certificate, is 'client' too broad? I know we are calling that in the UI so I'm ok with what you have here.  But I do wonder if we should be more specific with something like 'mtlsCertificateContent'

Reply:
I think we'd still want to call it a "client certificate" as it better describes the type of certificate thats being used. mTLS is just the method of mutual authentication between the client and server. I don't believe there is explicitly an "mTLS certificate"

Reply:
Agreed with what Martin said. I believe alert has an always trust option? this means it can be one way as opposed to mtls

Reply:
Yes @ChomickiM  is correct.  With mTLS (mutual TLS) there are two certificates used.  The server certificate on the server to receive the HTTP request.  The client certificate which is on the client system that is initiating the HTTP request to the server.
The certificates in use are:

- client certificate
- server certificate
- any intermediate and root certificate authority certificates used to sign both the client and server certificates which validates that the client and server can trust one another.